app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -110,10 +115,23 @@ export async function importNmls(files: Array<File>, createGroupForEachFile: boo
       throw errors;
     }
 
+    const currentDatasetName = Store.getState().dataset.name;
+    const doDatasetNamesDiffer = importActionsWithDatasetNames
+      .map(el => el.datasetName)
+      .some(name => name != null && name !== currentDatasetName);
```

Comment:
An empty experiment name `name=""` will trigger the confirm. I'd say it would be better to add `&& name.length > 0` after the null check.